packages/postgres-database/src/simcore_postgres_database/migration/versions/22404057a50c_resource_tracker_service_runs_helpers_.py

```diff
@@ -0,0 +1,35 @@
+"""resource_tracker_service_runs helpers for missing heartbeat
+
+Revision ID: 22404057a50c
+Revises: 2a4b4167e088
+Create Date: 2023-10-25 19:17:29.928871+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "22404057a50c"
+down_revision = "2a4b4167e088"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "resource_tracker_service_runs",
+        sa.Column("service_run_status_msg", sa.String(), nullable=True),
+    )
+    op.add_column(
+        "resource_tracker_service_runs",
+        sa.Column("missed_heartbeat_counter", sa.SmallInteger(), nullable=False),
```

Comment:
are you sure that this migration will go through without issue? it is non nullable, so if there are already values in the table, these will actually be NULL or will there be a magic default? I see you have a default on the model but non here. is this in sync?

Reply:
Thanks, good catch! I missed that the default value was not propagated here from the model.